Mathlib/CategoryTheory/Limits/Final.lean

```diff
@@ -296,6 +296,14 @@ def isColimitWhiskerEquiv (t : Cocone G) : IsColimit (t.whisker F) ≃ IsColimit
   IsColimit.ofCoconeEquiv (coconesEquiv F G).symm
 #align category_theory.functor.final.is_colimit_whisker_equiv CategoryTheory.Functor.Final.isColimitWhiskerEquiv
 
+/-- When `F : C ⥤ D` is final, and `t : Cocone G` for some `G : D ⥤ E` is colimiting, then the
+cocone points of `t` and `t.whisker F` are isomorphic. -/
+def coconePointsIsoWhisker {t : Cocone G} (ht : IsColimit t) : t.pt ≅ (t.whisker F).pt where
```

Comment:
All the definitions `coconePointsIsoWhisker`, `conePointsIsoExtendCone`, `conePointsIsoWhisker`, `conePointsIsoExtendCone` are unnecessary because they could be obtained more easily as `Iso.refl _`!

However, we may indeed remove the two auxiliary lemmas like `colimit_pre_is_iso_aux`.

Reply:
Hah, of course! I'll remove them